app/string_to_id.rb

```diff
@@ -3,6 +3,7 @@ class StringToId
   # which has caused a TOC bug // ref https://trello.com/c/Re6fSBKj/24-change-internal-links-to-work-or-remove-internal-links
   # this implementation modified for our purposes from version at jch/html-pipeline
   # https://github.com/jch/html-pipeline/blob/master/lib/html/pipeline/toc_filter.rb
+  # We now use Kramdown instead of Redcarpet, but if it ain't broke, don't fix it.
```

Comment:
But...can we fix it?

Reply:
My comment was a bit facetious, I should have explained the value in leaving this alone 😅 Have now removed the comment in a separate commit which explains why things have moved on since this comment.

Reply:
Thanks! I think it's out-of-scope, but worth noting this class isn't tested, so it may be even less clear now why it's necessary. As a follow-up, it may help future devs to have a couple of tests that showcase the specific use cases for this method that we don't get from the library itself.

Reply:
It does have [some tests](https://github.com/alphagov/govuk-developer-docs/commit/77f176ff84f4059af86d75ae3b09ade622f46cdf#diff-1b36fb860ebf4c903d116690329a1a8eR41-R44), but I agree it's tested indirectly and needs a bit of digging to find!